packages/contracts-bedrock/scripts/deploy/VerifyOPCM.s.sol

```diff
@@ -338,6 +381,65 @@ contract VerifyOPCM is Script {
         return success;
     }
 
+    /// @notice Verifies that the immutable variables in the OPCM contract match expected values.
+    /// @param _opcm The OPCM contract to verify immutable variables for.
+    /// @return True if all immutable variables are verified, false otherwise.
+    function _verifyOpcmImmutableVariables(IOPContractsManager _opcm) internal view returns (bool) {
+        console.log("  Verifying OPCM immutable variables...");
+
+        bool success = true;
+
+        // Define the function names and corresponding environment variable names
+        string[] memory functionNames = new string[](4);
+        functionNames[0] = "superchainConfig";
+        functionNames[1] = "protocolVersions";
+        functionNames[2] = "superchainProxyAdmin";
+        functionNames[3] = "upgradeController";
+
+        string[] memory envVarNames = new string[](4);
+        envVarNames[0] = "EXPECTED_SUPERCHAIN_CONFIG";
+        envVarNames[1] = "EXPECTED_PROTOCOL_VERSIONS";
+        envVarNames[2] = "EXPECTED_SUPERCHAIN_PROXY_ADMIN";
+        envVarNames[3] = "EXPECTED_UPGRADE_CONTROLLER";
```

Comment:
We shouldn't need this anymore. 
Instead we can:
1. get an array from `_getOpcmGetters()`
2. iterate over that array
3. if an entry is not `"SKIP"`, then we should use the string to get read the address from the env.

Reply:
Solved in 2de2f02d47fb52247e66655e338e64413c42adfe by applying literally your comment